issue99_test.go

```diff
@@ -0,0 +1,78 @@
+package zmq4
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+)
+
+var (
+	wg     sync.WaitGroup
+	outMsg Msg
+	inMsg  Msg
+)
+
+func requester(t *testing.T) {
```

Comment:
instead of defining the globals above and these `requester`/`responder` functions, I'd define `func` literals directly inside the `TestIssue99` function.

Reply:
Why?